### DISCUSSION TITLE
電動車一定環保嗎?

### DISCUSSION THREAD

**Discussion by Joan0518** [1101-GS4719-C/Python-Community-GS4719-C#114] [General]
隨著近年來環保議題的熱潮，各國都在積極引進電動車市場，拚燃油車轉型，在評估電動車環保效益時，也必須考量該區域的發電結構。例如台灣主要發電為火力發電，以火力發電作為電來源的電動車，在環保意義上就本末倒置，另外在生產電動車與回收老舊燃油車時，也會產生大量的碳排放。

此外電動車還有以下發展困難點:
1. 轉型初期若充電站分布不密集，容易造成「里程焦慮」，減少購買意願
2. 電動車發展技術雖可達到V2G(將電動車的電回充到智慧電網中)，但若智慧電網發展不夠成熟，仍無法發揮理想化的功能
3. 電動車價格居高不下
---
新聞內容https://tw.news.yahoo.com/%E9%9B%BB%E5%8B%95%E8%BB%8A%E4%B8%80%E5%AE%9A%E6%B8%9B%E7%A2%B3%EF%BC%9F%E5%8F%B0%E7%81%A3%E3%80%8C%E9%AB%92%E9%9B%BB%E3%80%8D%E5%8D%A0-8-%E6%88%90%E7%92%B0%E4%BF%9D%E7%9B%AE%E6%A8%99%E9%9B%A3%E9%81%94%E6%88%90-034056894.html

**Comment by Jie2222** [comment]
[台灣推電動車真的環保嗎](https://e-info.org.tw/node/209681)
台灣電動車電池以鋰電池為主，開採、製造與廢棄過程如無法妥善，也會導致污染，且目前國內處理能力不足

**Comment by wax75397** [comment]
志祺七七有做過相關的影片，裡面有提到一些電動車的分析，分享給大家看
影片：
[https://www.youtube.com/watch?v=owWbX5za6ps&t=516s&ab_channel=%E5%BF%97%E7%A5%BA%E4%B8%83%E4%B8%83X%E5%9C%96%E6%96%87%E4%B8%8D%E7%AC%A6](url)

**Comment by Jen-Siang** [comment]
業者在銷售電動車時，多數只提電動車使用上有多麼環保方便，而不會提到一顆電池
怎麼製造，廢棄後如何處理，以至於人們常看到的只有電動車的優點，極少看到缺點

**Comment by chun1203** [comment]
感覺廢棄電池的也是一個值得探討的問題

**Comment by HSU-YU-JEN** [comment]
很同意這篇文章，電動車使用的鋰離子電池，裡面用了許多貴重金屬原料，而這些原料從取得到加工生產，污染性都很高。

**Comment by EricAgumonJou** [comment]
關於廢棄電池處理資料：
根據台灣電池協會的資料指出，芬蘭再生能源公司Fortum已透過全新回收技術，透過二氧化碳濕法冶金技術從電池中回收鈷、鋰、錳和鎳等電池重要金屬，讓其他電池製造商可重新使用這些二次金屬。其開發的新技術也將電動車電池材料回收率從50%提高到80%，只是技術尚未普及。

**Comment by Jie2222** [comment]
歐洲國家雖然有比較成熟的技術，但是回收電池的**勞動力**成本高，很難形成好的經濟效益市場

**Comment by Isongzhe** [comment]
感覺這樣的議題很有深度。之前我去中興環工面試時，他就問我類似的問題。說台灣的電力主要來自火力發電，那你認為電動機車跟一般的二行程機車誰比較環保，為什麼?

**Comment by cody910120** [comment]
電動車不一定環保，但一定很貴

**Comment by 107602505** [comment]
看完這篇文章我已經無法很篤定的說電動車一定比油料車環保了，確實吃什麼電很重要，若要真的做到環保、減碳，發電過程必須是乾淨的，不過無論是電動車還是燃油車，都應同步朝減碳邁進。

**Comment by vvvioletta** [comment]
台灣也有一些電池原料廠商，像美琪瑪之類的，已經在開發貴金屬回收的業務，應該可以幫助建立台灣自己的電動車電池回收的生態鏈

**Comment by YuHengChen54** [comment]
之前在研究Gogoro就發現，其實背後發電廠也是燃煤發電，也造成相當程度的空氣汙染
